test/common-utils/scenarios.json

```diff
@@ -126,6 +126,16 @@
             }
         }
     },
+    "configure_zsh_no_template_first_step": {
+        "image": "debian:bullseye",
+        "remoteUser": "vscode",
+        "features": {
+            "common-utils": {
+                "installZsh": true,
+                "installOhMyZshConfig": false
+            }
+        }
+    },
```

Comment:
Looks like there is a failing check, see https://github.com/devcontainers/features/actions/runs/5849488185/job/15890491045?pr=653#step:4:14862